### ISSUE TITLE
Use assets bundler and proper dependency manager

### ISSUE THREAD

**Issue by blazeu** [coala/landing-frontend#235]
See https://medium.freecodecamp.org/javascript-modules-part-2-module-bundling-5020383cf306

### TL;DR
- Having many <script> and <style> tags sucks and makes loading slow
- Combine all assets into a single `.js` & `.css` file ([Webpack](https://webpack.js.org/) preferred)
- Since we're probably gonna use ES2015 modules to import modules/scripts, we can write **all** the codes in ES2015+ and have [Babel](http://babeljs.io/) + [Webpack](https://github.com/babel/babel-loader) integrations to transpile the code into current version of JavaScript that is widely supported by current browsers
- Yarn/NPM(v5) is currently the [recommended dependency manager](https://bower.io/blog/2017/how-to-migrate-away-from-bower/) for front-end (previously only for backend Node.js)
